src/WingetCreateCLI/TokenHelper.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCLI
+{
+    using System;
+    using System.Runtime.InteropServices;
+    using System.Text;
+    using Windows.Win32;
+    using Windows.Win32.Foundation;
+    using Windows.Win32.Security.Credentials;
+
+    /// <summary>
+    /// Provides functionality for caching and retrieving the GitHub OAuth token.
+    /// </summary>
+    public static class TokenHelper
+    {
+        // Windows credentials manager
+        private const string CredTargetName = "git:https://aka.ms/winget-create";

```

Comment:
```suggestion
        private const string CredTargetName = "winget-create:GitHub [read,pr write]";
```
We shouldn't step into the namespace of the Git Credential Manager for Windows, which I believe is what is creating the `git:` credentials.  Instead, use our own namespace (`winget-create:`) and a reasonably accurate description of what the token is so that if we need to expand the scope later, we have a pattern for a new token name.

It also helps to self-document the credential in the event that users want to manage it directly (they know where it came from and what it is for).

Reply:
Renamed to `winget-create:GitHub [repo]`